src/Employer/Esfa.Recruit.Azure.Resources/azuredeploy.json

```diff
@@ -755,6 +755,26 @@
               {
                 "name": "PasAccountApiConfiguration:Tenant",
                 "value": "[parameters('PasAccountApiConfiguration').Tenant]"
+              },
```

Comment:
Could you update the developer setup page on Confluence to include this in user secrets as well please for Provider.Web

Reply:
👍 I've done this now.